scripts/tests/cirque_tests.sh

```diff
@@ -33,43 +33,35 @@ BOLD_YELLOW_TEXT="\033[1;33m"
 BOLD_RED_TEXT="\033[1;31m"
 RESET_COLOR="\033[0m"
 
-function __flask_clean() {
-    flask_pid=$(ps aux | grep "[f]lask run" | grep -v "sudo" | awk '{print $2}' | sort -k2 -rn)
```

Comment:
The `grep -v "sudo"` part is still needed, no?  Below we do:
```
    sudo FLASK_APP='cirque/restservice/service.py' \
        PATH="$PATH":"$REPO_DIR"/third_party/cirque/repo/openthread/output/x86_64-unknown-linux-gnu/bin/ \
        python3 -m flask run >"$LOG_DIR/$CURRENT_TEST/flask.log" 2>&1
```
So there will be a `sudo` command line with `"flask"` in it...

Reply:
I'm trying to whack sudos

Reply:
updated